src/renderer/utils/installUdevRules.js

```diff
@@ -0,0 +1,44 @@
+// -*- mode: js-jsx -*-
+/* Chrysalis -- Kaleidoscope Command Center
+ * Copyright (C) 2020  Keyboardio, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import sudo from "sudo-prompt";
+import path from "path";
+import { getStaticPath } from "../config";
+
+const installUdevRules = async devicePath => {
+  const rules = path.join(getStaticPath(), "udev", "60-kaleidoscope.rules");
+  return new Promise(async (resolve, reject) => {
```

Comment:
We should, at the very lease, check for an existing file and move it to to .bak (if that won't get picked up by udev too)

Also, we should log that we're doing the install

Reply:
udev only picks up `.rules` files, so we can move any existing ones to `.orig` or something along those lines. Is logging to the dev console sufficient?

Reply:
Right now, logging to dev console is all we've got, right? Then let's do that.